zuul.d/projects.yaml

```diff
@@ -662,15 +656,15 @@
     default-branch: devel
     templates:
       - system-required
-      - ansible-python-jobs
       - ansible-python3-jobs
+      - runner-python2-jobs
```

Comment:
Zuul's lint job says that this isn't being found. Is that just because `project-templates.yaml` is a new file, added in this PR?

Reply:
This isn't correct, an can be removed. there is no project-template created.